README.md

```diff
@@ -48,6 +48,19 @@ Current collection
   - config:
     - style rules are in-repo
     - '-g .': Use git to auto-find .md files in '.'
+- [pytype][]
+  - docker
+  - Fedora 34
+  - Py 3 < 3.10, Fedora default (3.9.x)
+  - pip
+    - pytype (latest)
+  - config
+    - defaults
+    - '-d import-error': ignore imports (pytype will fail (in GitHub workflows)
+      when checking for third party imports, unless they're explicitly
+      installed)
```

Comment:
This might be another argument for doing type-checks with `tox` or something like that instead of using custom github actions. But these actions should still be a good default when you don't want to set up tox in a repo